projects/Mallard/src/paths/index.ts

```diff
@@ -40,10 +41,16 @@ const editionDir = async () => {
     const edition = await getSelectedEditionSlug()
     return `${issuesDir}/${edition}`
 }
+const edtionsDirList = async (): Promise<string[]> => {
+    return defaultRegionalEditions.map(reg => {
+        return `${issuesDir}/${reg.edition}`
+    })
+}
 
 export const FSPaths = {
     issuesDir,
     editionDir,
+    edtionDirList: edtionsDirList,
```

Comment:
Worth either adding or removing the `s` to benefit from the object shorthand syntax